docs/run-platform-server/prerequisites.mdx

```diff
@@ -3,56 +3,47 @@ title: Prerequisites
 sidebar_position: 10
 ---
 
-Horizon only has one true dependency: a PostgreSQL server that it uses to store data that has been processed and ingested from Stellar Core. **Horizon requires PostgreSQL version >= 9.5**.
+The Horizon service is responsible for synchronizing with the Stellar network and processing ledger data. We recommend reading the [configuring](./configuring.mdx) section as a primer on scope of Horizon's services before diving into compute pre-requisites.
 
-As far as system requirements go, there are a few main things to keep in mind. Starting from version 2.0, Horizon must be run as a standalone service. A full Horizon build consists of three functions:
+The Horizon service can be [installed](./installing.mdx) on bare metal or a virtual machine. It is natively supported on both Linux and Windows operating systems.
 
-1. **ingesting data** _from_ the decentralized Stellar network,
-1. **submitting transactions** _to_ the network, and
-1. **serving** API requests
+**Single Instance Deployment model**
 
-The first two happen through _Captive Core_, a pared down, non-validating version of Stellar Core packaged directly into Horizon.
+At a minimum, following the [Single Instance Deployment model](./configuring.mdx#single-instance-deployment) will require two compute instances:
 
-With these three functions in mind, you can also run Horizon in two different ways: **real-time ingestion** and **historical catch-up**:
+- one compute instance for Horizon
+- one compute instance for PostgreSQL server.
 
-- _Real-time ingestion_ is an “online” process: it involves keeping in sync with the live Stellar network state and digesting ledger data into a holistic view of the network. If you just want function (3) from above, you still need to do this.
+To run Horizon, we recommend a single compute profile which is applicable for single or multiple instances, the minimum hardware specifications are:
 
-- _Historical catch-up_ is an “offline” process: it lets you look into the past and catch up your Horizon instance to a given retention period (e.g. 30 days of history). Because it’s typically done offline and a one-time process, you can dedicate more compute power and configure parallel workers to catch up faster.
+**_Horizon compute instance:_**
 
-### Historical Catch-up
+- CPU: x cores
+- RAM: x GB
+- Storage: SSD with a capacity of x GB
 
-In this scenario, the hardware specifications are more demanding than what is necessary for the day-to-day operation of real-time ingestion, but catch-up only needs to occur once.
+**_PostgreSQL database server compute instance:_**
```

Comment:
this is currently a placeholder, we will state min compute requirements for postgres here correct? This seems like good place to also stipulate the postgres server version requirements, such as from team discussion. Just for consideration, maybe we relocate the low level postgres server configs mentioned in [Configuring](https://github.com/stellar/stellar-docs/blob/partner_experience/docs/run-platform-server/configuring.mdx#L138), inline here instead, to keep all postgres/vendor specifics about the db here in one place, then rest of docs just refer to postgres more generically as the database. 

CC'ing @tamirms for postgres context